hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/AbstractJetInstance.java

```diff
@@ -133,11 +137,22 @@ public HazelcastInstance getHazelcastInstance() {
         return hazelcastInstance.getList(name);
     }
 
+    @Nonnull @Override
+    public <T> ITopic<T> getTopic(@Nonnull String name) {
+        return hazelcastInstance.getTopic(name);
+    }
+
     @Nonnull @Override
     public JetCacheManager getCacheManager() {
         return cacheManager;
     }
 
+    @Override
+    public <T> Observable<T> getObservable(@Nonnull String name) {
+        ITopic<ObservableBatch> topic = getTopic(name);
```

Comment:
Shouldn't we use some prefix here? Otherwise the namespace clashes with `ITopic`, and we conceal that.